internal/photoprism/mediafile.go

```diff
@@ -314,6 +320,113 @@ func (m *MediaFile) EditedName() string {
 	return ""
 }
 
+func (m *MediaFile) VideoFileName(jsonName string) (string, error) {
+	if m == nil {
+		return "", fmt.Errorf(
+			"mediafile: file is nil - you may have found a" +
+				" bug",
+		)
+	}
+
+	// Abort if the source media file does not exist.
+	if !m.Exists() {
+		return "", fmt.Errorf(
+			"mediafile: %s not found", clean.Log(m.RootRelName()),
+		)
+	} else if m.Empty() {
+		return "", fmt.Errorf(
+			"mediafile: %s is empty", clean.Log(m.RootRelName()),
+		)
+	}
+
+	// given the filename, read the json sidecar
+	jsonBytes, err := os.ReadFile(jsonName)
```

Comment:
Maybe we can add this logic to the existing ExifTool JSON parser?

https://github.com/photoprism/photoprism/blob/develop/internal/meta/json_exiftool.go

The field name of the embedded video file (if any) could be written to a new field you add to https://github.com/photoprism/photoprism/blob/develop/internal/meta/data.go, e.g. `meta.Data.EmbeddedVideo`.

All you need to do then is:
```go
if metaData := m.MetaData(); metaData.Error == nil {
  // your code
}
```